components/link/link.scss

```diff
@@ -9,13 +9,23 @@
 		text-decoration: none;
 		cursor: pointer;
 	}
-	&:hover,
-	&:focus,
-	&.d2l-link-focus {
+	&:hover {
 		color: $d2l-color-celestine-minus-1;
 		text-decoration: underline;
-		outline-width: 0;
 	}
+	&:focus-visible {
+		border-radius: 2px;
```

Comment:
I put the border-radius down here vs. Dave's original change had it at the root. This matches what the web component is doing, and isn't applying a radius when focus isn't present, which I felt was a bit cleaner.